components/action-mgt/org.wso2.carbon.identity.action.execution/src/main/java/org/wso2/carbon/identity/action/execution/impl/ActionExecutorServiceImpl.java

```diff
@@ -316,14 +322,14 @@ private ActionExecutionStatus processActionResponse(Action action,
             logErrorResponse(action, actionInvocationResponse);
         }
 
-        return new ActionExecutionStatus(ActionExecutionStatus.Status.FAILED, eventContext);
+        return new FailedStatus();
```

Comment:
Given that failed status is for client errors this should be error isn't it ?

Reply:
This is fixed with 148073eab6305adc3e26c71ca755a0550e805f54